### PR TITLE
Fix registrations displaying

### PR DESCRIPTION
**Pull request by ghost**
This should fix the registration problem #5589

### CODE REVIEW

File: app/helpers/statistics_helper.rb [hunk 1]
```diff
@@ -4,13 +4,21 @@
 
 module StatisticsHelper
   def registrations_status statistics
-    if statistics.open_registrations?
+    if !!statistics.open_registrations?
```
Comment (dimaursu) [inline app/helpers/statistics_helper.rb]: um? That's double negation, it's the same as the original.

File: app/helpers/statistics_helper.rb [hunk 2]
```diff
@@ -4,13 +4,21 @@
...
       I18n.t('statistics.open')
     else
       I18n.t('statistics.closed')
     end
   end
 
+  def registrations_status_class statistics
+    if !!statistics.open_registrations?
```
Comment (dimaursu) [inline app/helpers/statistics_helper.rb]: Same as before, double negation

File: app/helpers/statistics_helper.rb (hunk 1 above)
Comment (AugierLe42e) [inline app/helpers/statistics_helper.rb]: Yeah, problem is : function recieve a symbol, so, as is is not a fasly value, it is always considered true, even if registrations are closed. `!!` "operator" forces cast to boolean. Works well ;)

Comment (jhass) [inline app/helpers/statistics_helper.rb]: * If you can call `open_registrations?` on `statistics` that's certainly not a symbol.
* If  `open_registrations?` returns a symbol, `!!symbol` is always `true`.
* if `open_registrations?` returns nil or a symbol, nil is falsey and the symbol is truthy, so the `!!` is unneeded.

Comment (AugierLe42e) [inline app/helpers/statistics_helper.rb]: Well I can't explain but I can assure you that it works with and doesn't work without :/

Comment (jhass) [inline app/helpers/statistics_helper.rb]: Add `require "pry"; binding.pry` in the line before that and trigger the code path.

Comment (jhass) [inline app/helpers/statistics_helper.rb]: Or just add a `?` to https://github.com/diaspora/diaspora/blob/develop/app/presenters/statistics_presenter.rb#L34 to properly fix it.

Also add an RSpec test that triggers the bug.

File: app/presenters/statistics_presenter.rb [hunk 3]
```diff
@@ -31,7 +31,7 @@ def version
   end
 
   def open_registrations?
-    AppConfig.settings.enable_registrations
+    AppConfig.settings.enable_registrations?
```
Comment (AugierLe42e) [inline app/presenters/statistics_presenter.rb]: @jhass : Nope, [see by yourself](https://travis-ci.org/diaspora/diaspora/jobs/48826537#L447), it doesn't work at all, plus it makes 2 other tests to fail.

File: app/presenters/statistics_presenter.rb [hunk 4]
```diff
@@ -31,7 +31,7 @@ def version
   end
 
   def open_registrations?
-    AppConfig.settings.enable_registrations
+    AppConfig.settings.enable_registration?
```
Comment (AugierLe42e) [inline app/presenters/statistics_presenter.rb]: @jhass : Nope, [see by yourself](https://travis-ci.org/diaspora/diaspora/jobs/48867657#L453). It doesn't work, plus it makes to other tests to fail. :/

Comment (jhass) [inline app/presenters/statistics_presenter.rb]: Take a close look at the line you commented on.

Comment (AugierLe42e) [inline app/presenters/statistics_presenter.rb]: Ok, are you willin to continue asking me riddles or will you eventually help me ?
[You](https://github.com/diaspora/diaspora/pull/5599#discussion_r23799476) told me tu put a `?` at the end of that line and I did and it doesn't work. So I'm sorry if I'm missing something but right now, I've got better to do than playing puzzles.

Comment (jhass) [inline app/presenters/statistics_presenter.rb]: You didn't add a `?`, you replaced the `s` with it.

Comment (AugierLe42e) [inline app/presenters/statistics_presenter.rb]: Hmmf... You are right, sorry.

**Comment by AugierLe42e** [comment]
Ok, I think it can be merged now.

**Comment by jhass** [comment]
Thanks!

**Comment by AugierLe42e** [comment]
Thanks to you.

**Pull request merged**